README.md

```diff
@@ -1,186 +1,80 @@
-# mock-spacestation
+# Mock-Spacestation
```

Comment:
re-enabling the `markdownlint` extension raised a lot of errors. Happy to help format this document if you'd like 👍.